CodeSignESRP.targets

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+	<UsingTask TaskName="ESRPSignTask" AssemblyFile="$(CISignRepoPath)\tools\sdkbuildtools\tasks\MS.Az.Sdk.OnPremise.Build.Tasks.dll" />
+
+	<PropertyGroup>
+		<!-- CISignRepo is an environment variable that points to ci-signing repo clone -->
+		<CISignRepoPath>$(CISignRepo)</CISignRepoPath>
+	</PropertyGroup>
+   
+	<Target Name="CodeSignBinariesESRP" DependsOnTargets="RestoreNugetPackages;BuildMsBuildTask">
```

Comment:
I don't think we need the ESRP prefix - since we are replaing CodeSign, let's just name these tasks CodeSignBinaries and CodeSignInstaller